#!/usr/bin/env python
PACKAGE = "test_pkg"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("deltatime",        double_t, 0, "A double parameter", 0.1, 0.001, 5.0)
gen.add("v_min",            double_t, 0, "A double parameter", -0.2, -1.0, 0.0)
gen.add("v_max",            double_t, 0, "A double parameter", 0.2, 0.0, 1.0)
gen.add("v_delta",          double_t, 0, "A double parameter", 0.05, 0.001, 0.3)
gen.add("omega_min",        double_t, 0, "A double parameter", -1.0, -3.0, 0.0)
gen.add("omega_max",        double_t, 0, "A double parameter", 1.0, 0.0, 3.0)
gen.add("omega_delta",      double_t, 0, "A double parameter", 0.1, 0.001, 1.0)

exit(gen.generate(PACKAGE, "local_planner", "local_planner"))